lint/settings.py

```diff
@@ -81,25 +81,29 @@ def get_settings_objects():
     for name in sublime.find_resources("SublimeLinter.sublime-settings"):
         try:
             yield name, util.load_json(name, from_sl_dir=False)
-        except IOError as ie:
+        except IOError:
             util.printf("Settings file not found: {}".format(name))
-        except ValueError as ve:
+        except ValueError:
```

Comment:
Could there be any useful information in this exception?

Reply:
The `IOError` is a super rare edge case. It would mean that either between looking for files at line 81 and reading one at line 83 the file has become unavailable. 

`ValueError` definitely contains valuable info, e.g. if you're missing a comma. However, if you make such an error in your settings file, SublimeText already throws you a popup explaining what's wrong. 

We should only concern ourselves with validity of user settings in terms of it containing values we understand. Being able to load and parse user settings is ST's job and it handles all of that already.

Reply:
Good point. Didn't think of ST already raising errors for invalid settings files.